pkgs/os/os.sh

```diff
@@ -107,31 +144,54 @@ _print() {
           ;;
       esac
     done
```

Comment:
this loop is missing a default case to still consume arguments when they are invalid

Reply:
done